app/src/main/java/com/rentminder/MainActivity.kt

```diff
@@ -194,9 +181,7 @@ fun EditBillAmounts() {
                             focusManager.clearFocus()
                         }
                     ),
-                    modifier = Modifier
-                        .width(100.dp)
-                        .height(55.dp),
+                    modifier = Modifier.width(100.dp).height(55.dp),
                     textStyle = TextStyle.Default.copy(fontSize = 18.sp)
```

Comment:
made the code more efficient.
https://kotlinlang.org/docs/coding-conventions.html#configure-style-in-ide